src/app/starredentries/starredentries.component.html

```diff
@@ -81,12 +82,42 @@
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool">
+        <mat-tab>
+          <ng-template mat-tab-label>
+            <div fxLayoutGap="0.5rem">
+              <img class="notebook-icon" src="../../../assets/svg/sub-nav/notebooks.svg" alt="notebook logo" />
+              <span>Notebooks</span>
+              <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
+            </div>
+          </ng-template>
+          <div>
+            <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredNotebooks?.length === 0">
+              <mat-icon>warning</mat-icon>
+              You have no starred notebooks.
+            </mat-card>
+            <div fxLayout="column" fxLayoutGap="1rem" class="mb-2 mx-1">
+              <mat-card fxFlex *ngFor="let notebook of starredNotebooks; let last = last" class="m-1 pb-3">
+                <ng-container
+                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, isAppTool: false, isNotebook: true }"
+                ></ng-container>
+              </mat-card>
+            </div>
+          </div>
+        </mat-tab>
+
+        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool" let-isNotebook="isNotebook">
```

Comment:
Recommend that we pass the proper `entryType` into the template and dispense with the `isAppTool` and `isNotebook`.